packages/notification-center-vue/src/lib/notification-center-component/BellButton.vue

```diff
@@ -0,0 +1,153 @@
+<script lang="ts">
+import { colors as themeColors } from '@novu/notification-center';
+
+export default {
+  data() {
+    const color = this.colors.unseenBadgeColor;
+    const regularColor = typeof color === 'string';
+
+    return {
+      gradientDot: {
+        stopColor: regularColor ? (color as string) : (color as ISvgStopColor).stopColor,
+        offsetStopColor: regularColor ? (color as string) : (color as ISvgStopColor).stopColorOffset,
+      },
```

Comment:
calculate the gradient dot color stops, the logic was copied from the React package